README.md

```diff
@@ -36,6 +36,17 @@ Includes [Jenkins Pipeline](https://github.com/jenkinsci/workflow-plugin) suppor
 slackSend color: 'good', message: 'Message from Jenkins Pipeline'
 ```
 
+Additionally you can pass a JSONArray as a String in order to send complex messages
+
+```
+JSONArray jsonArray = new JSONArray()
+    .
+    .
+    .
```

Comment:
Rather than `.` please provide a real usage example so users not experienced with Groovy can see how to use it.